### PR TITLE
add a comment to manager module

### LINKED ISSUE #1: Uncommented blocks of code
`manager.py` has large blocks of code in, for example in the function `manage_customer` with no comments.

### PR DESCRIPTION
**Pull request by escalonn**
partially fix #1

### CODE REVIEW

**Comment by escalonn** [comment]
@SWHarrison ready for review

File: manager.py [hunk 1]
```diff
@@ -172,6 +172,7 @@ def manage_bank(bank_name):
 """
 def manage_customer(customer):
 
+    # user input should be one of the four choices (menu below)
```
Comment (escalonn) [inline manager.py:175]: give more context?

**Review by escalonn — COMMENTED** [review]
what about the rest of the file nick

**Comment by SWHarrison** [comment]
Not using due to test example in training

**Pull request closed without merging**